submission-scripts/script-templates/magpie-hive

```diff
@@ -0,0 +1,71 @@
+#############################################################################
+##Hive configurations
+#############################################################################
+#
+# Should Hive be run
+#
+# Specify yes or no.  Defaults to no.
+#
+export HIVE_SETUP=yes
+
+# Version
+
+export HIVE_VERSION="HIVEVERSIONDEFAULT"
+
+# Path to your Hive build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+# Ensure the build matches the Hadoop/HDFS version this will run against.
+#
+export HIVE_HOME="HIVEDIRPREFIX/base/apache-hive-${HIVE_VERSION}-bin"
+
+# Set the port for Hive to listen on
+
+export HIVE_PORT="11112"
+
+# Set HADOOP_HOME (IF NOT DONE ABOVE)
+# Hive uses Hadoop, so:
+#    you must have Hadoop in your path OR
+#    export HADOOP_HOME=<hadoop-install-dir>
+#
+# export HADOOP_HOME="${_PREFIX}/base/hadoop-${HADOOP_VERSION}"
+
+# need ANT_HOME configured for Hive web interface
+export ANT_LIB="HIVEDIRPREFIX/base/apache-ant-1.9.9/lib"
+
+# Path to store data local to each cluster node, typically something
+# in /tmp.  This will store local conf files and log files for your
+# job.  If local scratch space is not available, consider using the
+# MAGPIE_NO_LOCAL_DIR_DIR option.  See README for more details.
+
+export HIVE_LOCAL_DIR="LOCALDIRPREFIX/hive"
+export HIVE_METASTORE_DIR="LUSTREDIRPREFIX/hive/database/postgres/data"
+export HIVE_DEFAULT_METASTORE_DIR="LUSTREDIRPREFIX/hive/metastore"
+
+# Configurations for Postgres, used as a backend for Hive
+export POSTGRES_HOME="POSTGRESDIRPREFIX/base/pgsql"
```

Comment:
if I'm reading online about Hive correctly, perhaps there should be some type of "HIVE_METASTORE_TYPE", which can be set to mysql, postgresql, or oracle.  We will only support postgresql for now.

Then there would be configs for

HIVE_POSTGRES_HOME, HIVE_POSTGRES_DATA_DIR

Reply:
I think this is a great idea for future updates.  The only reason I went with PostgreSQL was because I was already using it in other places so it was easy/familiar.  I don't see why we couldn't make future updates for MySQL, etc.